plugin-nifi-registry/src/main/java/org/apache/ranger/services/nifi/registry/client/NiFiRegistryClient.java

```diff
@@ -50,30 +51,27 @@
  * Client to communicate with NiFi Registry and retrieve available resources.
  */
 public class NiFiRegistryClient {
-
-    private static final Logger LOG = LoggerFactory.getLogger(NiFiRegistryClient.class);
-
     static final String SUCCESS_MSG = "ConnectionTest Successful";
     static final String FAILURE_MSG = "Unable to retrieve any resources using given parameters. ";
-
-    private final String url;
-    private final SSLContext sslContext;
+    private static final Logger LOG = LoggerFactory.getLogger(NiFiRegistryClient.class);
```

Comment:
To be consistent, move instantiation of LOG to top of class definition. Please review all other occurrences as well.

Reply:
Missed this, thanks!